### ISSUE TITLE
Program displays hidden tabs.

### ISSUE THREAD

**Issue by celnahal** [bjarkeeck/PathOfExileStashSorter#3]
The tabs in json at the bottom of the call to get-stash-items has a member called "hidden" see below. For whatever reason I have 11 tabs (I really only have access in game to 10). Right now the program will just display this extra "1" tab. It wont crash, but is not quite right.

It was an easy fix on my end. 

In Models.cs under the "Tab" class I added 

[JsonProperty(PropertyName = "hidden")]
public bool hidden { get; set; }

In PoeConnector.cs in the FetchTabs method I changed

List<Tab> tabs = stash.Tabs;
to
List<Tab> tabs = stash.Tabs.Where(t => !t.hidden).ToList();

{
      "n": "1",
      "i": 1,
      "hidden": true,
      "selected": false,
      "colour": {
        "r": 124,
        "g": 84,
        "b": 54
      },
.
.

**Comment by bjarkeeck** [comment]
Yeah, i had this problem aswell where an extra tab named "1" would show up. I didn't investigate much i just added this lazy fix when downloading the tabs for a league:
            var t = Tabs.FirstOrDefault();
            if (t != null && t.Name.Trim() == "1")
                Tabs.Remove(t);

Good job on spotting the json hidden member! That's deffently a better solution, will fix this tomorrow or so. And thanks again for helping out man, really appreciate it :)
